src/platform/EFR32/OTAImageProcessorImpl.h

```diff
@@ -67,6 +67,12 @@ class OTAImageProcessorImpl : public OTAImageProcessorInterface
     OTADownloader * mDownloader;
     OTAImageHeaderParser mHeaderParser;
     const char * mImageFile = nullptr;
+
+#define ALIGNMENT_BYTES 64
```

Comment:
will this ALIGNMENT_BYTES escape the header? the name is very generic. I think `static constexpr size_t kAlignmentBytes = 64;` may be preferable and more type safe and will not escape beyond class.

Reply:
Fixed